### PR TITLE
Fix cmds running against directories with spaces

### PR DESCRIPTION
**Pull request by XScorpion2**
This fixes the build commands when running in directories with spaces.

### CODE REVIEW

File: EarTrumpet/prebuild.ps1 [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 $PackageManifestPath = "$PSScriptRoot\..\EarTrumpet.Package\Package.appxmanifest"
 
-$xml = [xml](Get-Content $PackageManifestPath)
+$xml = [xml](Get-Content "$PackageManifestPath ")
```
Comment (riverar) [inline EarTrumpet/prebuild.ps1]: Can you trim that extra space on this line?

Comment (XScorpion2) [reply EarTrumpet/prebuild.ps1]: Already ahead of you. I trimmed that extra space in the second commit https://github.com/File-New-Project/EarTrumpet/pull/525/commits/9704823165b6833cea52578ac795b700495ab87b

**Comment by riverar** [comment]
Please create a fresh PR off dev, thanks!

**Comment by riverar** [comment]
@XScorpion2 Hey there, any luck? I can check in the change but would prefer you get credit. Let me know if you need help.

**Comment by XScorpion2** [comment]
@riverar created new pr targeted against dev: https://github.com/File-New-Project/EarTrumpet/pull/552

**Pull request closed without merging**